cfme/utils/auth/__init__.py

```diff
@@ -86,7 +86,8 @@ def from_config(cls, prov_config, prov_key):
     def as_fill_value(self, user_type=None, auth_mode=None):
         """Basic implementation matches instance attributes to view form attributes"""
         class_attrs = [att.name for att in self.__attrs_attrs__]
-        include_attrs = [getattr(self.__class__, name)
+        # attr filter needs the Attribute object
+        include_attrs = [getattr(self.__class__.__attrs_attrs__, name)
```

Comment:
**required** please use `attr.fields_dict(type(self))` base and fetch the named elements from it

it seems most sensible to assign it to `class_attra` instead of that list

Reply:
Had a feeling you'd know a better way to pull these named attrs - I'll look at `fields_dict` use.